bindings/rust/wasmedge-sys/src/io/input2.rs

```diff
@@ -0,0 +1,26 @@
+use std::{marker::PhantomData, ops::Deref};
+
+use crate::{Value, WasmFnIO};
+
+pub struct Input2<A, B> {
+    inner: Vec<Value>,
+    phantom_a: PhantomData<A>,
+    phantom_b: PhantomData<B>,
+}
+
+impl<A, B> WasmFnIO for Input2<A, B> {
+    fn parameters() -> Vec<Value> {
+        match (std::any::type_name::<A>(), std::any::type_name::<B>()) {
+            ("i32", "i32") => vec![Value::I32(0), Value::I32(0)],
```

Comment:
Only two input parameters are supported?

Reply:
> Only two input parameters are supported?


First, still in drafting.  
Second, If you want to help, write a macro for this, but the API is still changing so far.
Less but not least,  be mannered because this is a draft and not asking for help so far.

Reply:
I know this is a draft, this part I had some ambiguities in the previous implementation, just communicating with you some questions.

Reply:
Is the exchange of ideas for realization not allowed here？

Reply:
I just want to learn from your realization

Reply:
I'd better wait until you're done with it. 😑

Reply:
Sorry about this.

I am struggling with the interface, so I am not ready to answer that when the interface is not ready.  
Besides, there is also some issue `Extraref` pointer for a function. 
Everything is still changing in a big, the CI help me exam my code under development in the draft, such that I can develop without fully setting up the local environment.
The part of the interface may be much ready now, the implementation will be `I1` to `I32` for the users and these will be enough for users.

And it is still good for me to discuss it, when this branch is ready to review, just in case we are discussing something interface under iterating and does not work at all, such that we can avoid wasting time and afford not on the draft under iterating.